components/kyma-environment-broker/internal/process/provisioning/inject_btp_operator_credentials_step.go

```diff
@@ -110,7 +110,7 @@ func (s *InjectBTPOperatorCredentialsStep) createOrUpdateSecret(k8sClient client
 	updateSecretData(&clusterSecret, parametersBasedSecret)
 	err = k8sClient.Update(context.Background(), &clusterSecret)
 	if err != nil {
-		log.Error("failed to update the secret for BTP Manager")
+		log.Errorf("failed to update the secret for BTP Manager: %w", err)
```

Comment:
when logging wrapping is not needed using %w, just use %s

Reply:
In the next commit.